Gruntfile.coffee

```diff
@@ -0,0 +1,339 @@
+module.exports = (grunt) ->
+  # Project configuration
+  grunt.initConfig
+    pkg: grunt.file.readJSON 'package.json'
+    assetsPath: 'assets'
+    webAssetsPath: 'web/assets'
+
+    # Concurrent configuration
+    concurrent:
+      syncFiles: [
+        'sync:coffee'
+        'sync:fonts'
+        'sync:images'
+        'sync:js'
+        'sync:sass'
+      ]
+      syncImagesAndJs: [
+        'sync:images'
+        'sync:js'
+      ]
+      generateAssets: [
+        'generateFonts'
+        'generateCssForProduction'
```

Comment:
Je gebruikt zowel "prod" als "Production" in deze file, ik zou voor een consistente benaming gaan (en imo is production beter).